_examples/complete/config/override-certification-manager.yaml

```diff
@@ -0,0 +1,24 @@
+## Node affinity for particular node in which labels key is "Infra-Services" and value is "true"
+
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: "eks.amazonaws.com/nodegroup"
+          operator: In
+          values:
+          - "critical"
+
+extraArgs: []
```

Comment:
If not using `extraArgs:` then it can be removed from here

Reply:
updated code as per suggestions